torch_ttnn/passes/lowering/target_wrappers.py

```diff
@@ -16,3 +16,71 @@ def repeat(t, sizes):
 @torch.fx.wrap
 def pack_to_tuple(*args):
     return tuple(args)
+
+
+@torch.fx.wrap
+def move_to_host(device_tensor, layout):
+    host_tensor = ttnn.from_device(device_tensor)
+    return ttnn.to_layout(host_tensor, layout)
+
+
+@torch.fx.wrap
+def conv2d(
+    input_tensor,
+    weight_tensor,
+    bias_tensor,
+    batch_size,
+    in_channels,
+    out_channels,
+    input_height,
+    input_width,
+    kernel_size,
+    stride,
+    padding,
+    dilation,
+    groups,
+    device,
+    transposed,
+    output_padding=None,
+):
+    if transposed:
+        conv_config = ttnn.Conv2dConfig(
+            shard_layout=ttnn.TensorMemoryLayout.HEIGHT_SHARDED,
+        )
+        output_tensor, _, _, _, _ = ttnn.conv_transpose2d(
```

Comment:
just fyi, I think other outputs are pre-processed inputs, which can be reused when more convs are called on same inputs.

Reply:
IMO it will be eaiser to pre-process inputs and cache them if ttnn can expose API to do that instead of doing inside `ttnn.conv2d` automatically, so we can control when to pre-process and cache the inputs

If such separation is not possible in short term and we need to start optimizing for performance, then we can try to cache the other outputs here and use them later.

Reply:
Such separation is possible. The PR to enable that is in progress. I will inform once it is completed.